internal/servers/controller/handlers/hosts/host_service.go

```diff
@@ -265,6 +257,9 @@ func (s Service) UpdateHost(ctx context.Context, req *pbs.UpdateHostRequest) (*p
 		idActions := idActionsTypeMap[host.SubtypeFromId(req.GetId())]
 		outputOpts = append(outputOpts, handlers.WithAuthorizedActions(authResults.FetchActionSetForId(ctx, h.GetPublicId(), idActions).Strings()))
 	}
+	if outputFields.Has(globals.HostSetIdField) {
+		outputOpts = append(outputOpts, handlers.WithHostSetIds(h.GetSetIds()))
+	}
```

Comment:
Why is UpdateHost treated differently from ReadHost or ListHosts regarding the outputFields?

Reply:
Good question. Before I wrote this if case, the function had nothing to append the host set ids option to proto. What determines whether a field needs an if case or is just appended by default?